packages/core/src/api/exporters/markdown/__snapshots__/codeBlock/defaultLanguage/markdown.md

```diff
@@ -1,3 +1 @@
-```javascript
-console.log('Hello, world!');
-```
+    console.log('Hello, world!');
```

Comment:
Not sure we can really do much about it either (since the HTML -> MD conversion is not handled by us), but this seems to be missing the code block formatting.

Reply:
Yea, I spent like 30 minutes looking into it. It does not want to properly convert it if it does not have a language specified

Reply:
I will just output language-text, I don't see the harm in it

```text
This example proves it should work fine
```